modules/tools/rosbag/stat_acc.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+###############################################################################
+# Copyright 2017 The Apollo Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+"""
+Stat acc in time sequence.
+Usage:
+    ./stat_acc.py bag1 bag2 ...
+"""
+
+import sys
+import os
+import datetime
+
+from rosbag.bag import Bag
+
+
+kChassisTopic = '/apollo/canbus/chassis'
+kLocalizationTopic = '/apollo/localization/pose'
```

Comment:
Looks like kLocalizationTopic and knano not needed?

Reply:
recommend use modules/tools/common/message_manager.py